opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -91,69 +105,25 @@ def test_collect(self):
             second_sum.start_time_unix_nano, first_sum.start_time_unix_nano
         )
 
-
-class TestAsynchronousSumAggregation(TestCase):
-    def test_instrument_monotonicity_awareness(self):
+    def test_collect_cumulative(self):
         """
-        `AsynchronousSumAggregation` is aware of the instrument monotonicity
+        `SynchronousSumAggregation` collects sum metric points
         """
 
-        asynchronous_sum_aggregation = AsynchronousSumAggregation(True)
-        self.assertIsInstance(
-            asynchronous_sum_aggregation,
-            _InstrumentMonotonicityAwareAggregation,
+        synchronous_sum_aggregation = SumAggregation(
```

Comment:
```suggestion
        sum_aggregation = SumAggregation(
```

And can you add a case for when the SumAggregation with async instrument returns None?

Reply:
Added a test case